### PR TITLE
add LICENSE to py_wheel

### PR DESCRIPTION
**Pull request by 6fj**

### CODE REVIEW

**Review by rivertalk — APPROVED** [review]
LGTM

File: ppu/BUILD.bazel [hunk 1]
```diff
@@ -24,6 +22,22 @@ package(default_visibility = ["//visibility:public"])
 
 PPU_VERSION = "0.0.4"
 
+LICENSE = """
+# Copyright 2021 Ant Group Co., Ltd.
```
Comment (zhouaihui) [inline ppu/BUILD.bazel:26]: `2021` -> `2022`

**Review by zhouaihui — APPROVED** [review]
LGTM

**Review by anakinxc — APPROVED** [review]
LGTM

**Pull request merged**